app/screens/channel/channel.tsx

```diff
@@ -136,6 +138,11 @@ const Channel = ({channelId, componentId, displayName, isOwnDirectMessage, membe
                         forceQueryAfterAppState={appState}
                     />
                 </View>
+                <PostDraft
+                    channelId={channelId}
+                    scrollViewNativeID={channelId}
+                    accessoriesContainerID={ACCESSORIES_CONTAINER_NATIVE_ID}
```

Comment:
This is a part that I haven't yet understood completely (the KeyboardTrackingView component), so not sure if we ever want to have different accesoriesContainerID for different PostDrafts. @enahum ?

Reply:
Ok let me explain.

There is a prop called `keyboardDismissMode` on lists (ScrollView, VirtualizedList, FlatList, SectionList), on iOS this prop can be set to `interactive` which means the keyboard will follow the finger dragging the keyboard up/down.

With React Native (until probably 0.68.0) when you use the interactive keyboard dismissal, they keyboard does display on top of the list covering the content, in our case, as the list is inverted, is covering the most recent posts.

There is also something called `InputAccessoryView` that allows you to attach a component just on top of the keyboard and it will follow the keyboard opening/closing and matching the animation, this for example would be used to keep the post draft always on top of the keyboard.

As I mentioned, today there is no support for what we want to accomplish (maybe with 68) which is to have the content of the list to adapt to the keyboard being opened/closed and interactive, this is where `KeyboardTrackingView` comes in, this is a module that allows us to keep the content of the list on top of the keyboard at all times as well as keeping the post draft following the keyboard. We accomplish this by telling to that module which is the ScrollView on which the content is being affected (this is done by our patch as initially this module was created because of the lack of support for `InputAccessoryView`). In order for the module to know which list to observe, we pass an ID as a prop so that the native code uses it to do the match. The reality is that we have a navigation stack that could have two screens, Channel -> Thread where both this screens have a list of posts and it's own post draft, that is why we then pass a prop to the PostDraft component telling it which is the `accessoriesContainerID` matching the view.

does it make sense?

Reply:
`accessoriesContainerID` will then be dynamic based on the navigation path and allowing it as a prop makes sense.